docs/source/11_tools_integration/02_ipython.md

```diff
@@ -103,9 +103,7 @@ The `context` variable allows you to interact with Kedro library components from
 With `context`, you can access the following variables and methods:
 
 - `context.project_path` (`Path`) - Root directory of the project
```

Comment:
Remind me what do we use `project_path` for? I remember we couldn't deprecate it because it was one of those annoying warnings you can't do anything about. Is it used in sth to do with the catalog?

Reply:
Yes it's used [for creating the catalog](https://github.com/quantumblacklabs/kedro/blob/master/kedro/framework/cli/catalog.py#L149) as well as [getting the catalog from the context](https://github.com/quantumblacklabs/kedro/blob/master/kedro/framework/context/context.py#L359-L361).